editor/src/document/document_message_handler.rs

```diff
@@ -103,15 +131,10 @@ impl MessageHandler<DocumentsMessage, &InputPreprocessor> for DocumentsMessageHa
 		use DocumentsMessage::*;
 		match message {
 			Document(message) => self.active_document_mut().process_action(message, ipp, responses),
-			SelectDocument(id) => {
-				assert!(id < self.documents.len(), "Tried to select a document that was not initialized");
-				self.active_document_index = id;
-				responses.push_back(
-					FrontendMessage::SetActiveDocument {
-						document_index: self.active_document_index,
-					}
-					.into(),
-				);
+			SelectDocument(index) => {
+				assert!(index < self.documents.len(), "Tried to select a document that was not initialized");
+				self.active_document_index = index;
+				responses.push_back(FrontendMessage::SetActiveDocument { document_index: index }.into());
```

Comment:
Okay, I added a comment here.

Reply:
Just fyi, this is because the message is serialized as json to get passed to js, but the js json parser does not support BigInts and instead just rounds it to the nearest floating point number